src/views/kbv/question.njk

```diff
@@ -0,0 +1,42 @@
+{% extends "base-form.njk" %}
+{% from "hmpo-text/macro.njk" import hmpoText %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+{% set gtmJourney = "kbv-hmrc - middle" %}
+
+{% set hmpoTitle = question.key %}
+
+{% block mainContent %}
+  <h1 class="govuk-heading-l"> {{ translate("pages.question.title") }} </h1>
+
+  {{ govukInsetText({
+    text: translate("pages.question.inset")
+  }) }}
+
+  {{ hmpoHtml(translate("pages.question.content")) }}
+
+  {% call hmpoForm(ctx) %}
+
+    {{ govukInput({
```

Comment:
I know we are using this to get the £ prefix but I think using a govuk rather than hmpo component is going to be an issue when it comes to validation/form saving which is done through their session model - @merlinc is that right?

Reply:
I dont believe the HMPO-component facilitate the `prefix` field. Is that something we will request they add? @merlinc.

Reply:
Hmm... there's a [currency filter](https://github.com/HMPO/hmpo-components/blob/c03a486b5a46343562dad4a8b19052f075e71165/lib/filters.js#L10) but that's for displaying content...

There's no specific mention of `prefix` or `suffix` in the hmpo-components repo - but they're not features needing to be changed by the hmpo-components for a page-form lifecycle so I wouldn't expect to see them.

If you look at [hmpo-number](https://github.com/HMPO/hmpo-components/blob/c03a486b5a46343562dad4a8b19052f075e71165/components/hmpo-number/macro.njk) it's just a wrapper around [hmpo-text](https://github.com/HMPO/hmpo-components/blob/c03a486b5a46343562dad4a8b19052f075e71165/components/hmpo-text/macro.njk) which is a wrapper around govukInput

It looks to my eyes like there's implicit passing through of all params, so I'd suggest trying to use a `hmpo-text` with whatever prefix you need and see what happens.

If that doesn't work, then we can make a PR in components to add in the passthrough properties of prefix/suffix

Warning: The current live docs are relating to govuk-frontend v5, so I had to go looking through the source code to find the [v4.8 govukInput](https://github.com/alphagov/govuk-frontend/blob/v4.8.0/src/govuk/components/input/template.njk) to see what was supported

Reply:
@blakeyp And yeah, the useful part of the hmpo-components is them wrapping the govuk components with the idea of "being inside a page" and "having error messages" - neither of which is given to us by the govuk-frontend

Reply:
We've raised a [PR](https://github.com/HMPO/hmpo-components/pull/157) (now merged) in hmpo-components to add `prefix`. While we await the next release we've replaced this with a plain `hmpoText` input to unblock us (ticket [PYIC-5045](https://govukverify.atlassian.net/browse/PYIC-5045) raised to pick this up at a later date)